company_dependent_flag/models/base.py

```diff
@@ -27,12 +27,45 @@ def _update_company_dependent_css(self, arch):
             if field_rec.company_dependent
         ]
         for field_name in cpny_dep_fields:
-            for field in arch.xpath(f"//field[@name='{field_name}']"):
-                classes = field.attrib.get("class", "").split(" ")
-                classes += self._get_company_dependent_css_class()
-                field.attrib["class"] = " ".join(set(classes))
+            for field in arch.getElementsByTagName("field"):
+                if field.getAttribute("name") == field_name:
+                    # Create a new div element
+                    div = arch.createElement("div")
+                    div.setAttribute("class", "o_row")
 
-    def _get_company_dependent_css_class(self):
-        """Inherit to apply your own class"""
+                    # Create a new span element
+                    span = arch.createElement("span")
+                    span.setAttribute(
+                        "class",
+                        "fa fa-lg fa-building-o title='Values set here are company-specific.'",
+                    )
```

Comment:
**issue**: `title='Values set here are company-specific.'` <-- this is not part of `class`, but a new attribute.

Reply:
Just fixed, thank you for pointing it out!